### PR TITLE
Add pa11y-ci

### PR DESCRIPTION
**Pull request by gemfarmer**
WIP 

#1896 

@mugizico Instead of waiting for Aidan to get to the concourse scanner, I thought we might consider solving this on our end first. With `pa11y-ci` I don't think it should take terribly long.

### CODE REVIEW

**Comment by gemfarmer** [comment]
Remaining work before this branch is merged:

- [ ] address hound's grievances (if they are reasonable)
- [ ] remove comments in sitemap.xml
- [ ] squash commits on branch. Refer to [thoughtbot](https://github.com/thoughtbot/guides/tree/master/protocol/git#write-a-feature) or [dolores landingham](https://github.com/18F/dolores-landingham-slack-bot/blob/develop/CONTRIBUTING.md#git-protocol) git protocol
- [ ] review and edit documentation that has been added. We might consider having @coreycaitlin @awfrancisco or @jamiealbrecht take a look, too!

**Comment by mugizico** [comment]
@gemfarmer the squashed commits could use some re-wording for clarity. Let's chat about it sometimes...

File: _posts/2016-03-18-how-user-archetypes-informed-the-draft-us-web-design-standards.md [hunk 1]
```diff
@@ -81,12 +81,12 @@ they defeat the purpose of a unified user experience.
 The Newbie is a designer or front end developer who is new to government
 and early in their career.
 
-#### Who they are
+<h4 id="who-they-are-2">Who they are</h4>
```
Comment (gemfarmer) [inline _posts/2016-03-18-how-user-archetypes-informed-the-draft-us-web-design-standards.md]: @elainekamlley this is an example of how we fixed the multiple header error

**Comment by gemfarmer** [comment]
🎉

**Pull request merged**